modules/web-console/e2e/testcafe/fixtures/queries/notebook.js

```diff
@@ -92,3 +93,38 @@ test('Sending a request', async(t) => {
         .click(confirmClearQueryDialog.confirmButton)
         .expect(paragraph.resultsTable._selector.exists).notOk();
 });
+
+test('Check z-indexes', async(t) => {
+    const paragraph2 = new Paragraph('Query1');
+
+    const LONG_CACHE_LIST = {
+        caches: {},
+        groups: []
+    };
+
+    for (let i = 0; i < 25; i++)
+        LONG_CACHE_LIST.caches['Cache' + i] = `${i}`;
+
+    await t.addRequestHooks(
+        t.ctx.ws = new WebSocketHook()
+            .use(
+                agentStat(FAKE_CLUSTERS),
+                cacheNamesCollectorTask(LONG_CACHE_LIST),
+                metadata(FAKE_METADATA)
+            )
+    );
+
+    await t.useRole(user).navigateTo(resolveUrl('/queries/notebooks'));
+    await notebooks.createNotebook('Foo');
+    await t.click(notebooks.getNotebookByName('Foo'));
+
+    await t.click(addQueryButton);
+    await t.expect(paragraph2.cacheList.visible).ok();
+    await t.pressKey('home');
```

Comment:
I'd replace this with `scrollToPageTop` helper function, similar to `scrollToPageBottom` that we already have. This will convey the intention more clear.